packages/wallets/src/evm/wallets/coinbase-wallet.ts

```diff
@@ -15,6 +15,11 @@ export class CoinbaseWallet extends AbstractBrowserWallet<{
 }> {
   connector?: TWConnector;
   coinbaseConnector?: CoinbaseWalletConnector;
+  static meta = {
+    iconURL:
+      "https://gateway.ipfscdn.io/ipfs/QmcJBHopbwfJcLqJpX2xEufSS84aLbF7bHavYhaXUcrLaH/coinbase.svg",
```

Comment:
please don't hard-code the full url, instead:
```suggestion
      "ipfs:///QmcJBHopbwfJcLqJpX2xEufSS84aLbF7bHavYhaXUcrLaH/coinbase.svg",
```

Reply:
and then where we display them we resolve it back on the configured storage (which by default will end up using this gateway but this lets users overwrite which gateway to pull it from)